modules/oadp-velero-cpu-memory-requirements.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-velero-cpu-memory-requirements_{context}"]
+= Velero CPU and memory requirements based on collected data
+
+The following recommendations are based on observations of performance made in the scale and performance lab. The backup and restore resources can be impacted by the type of plugin, the amount of resources required by that backup or restore, and the respective data contained in the persistent volumes (PVs) related to those resources.
+
+
+== CPU and memory requirement for configurations
+|===
+|Configuration types | ^[1]^ Average usage |^[2]^ Large usage |resourceTimeouts
+
+|CSI
+|Velero:
+
+CPU- Request 200m, Limits 1000m
+
+Memory - Request 256Mi, Limits 1024Mi
+
+|Velero:
+
+CPU- Request 200m, Limits 2000m
+
+
+Memory- Request  256Mi, Limits 2048Mi
+
+|
+
+|Restic
+|^[3]^ Restic:
+
+CPU- Request 1000m, Limits 2000m
+
+Memory - Request 16Gi, Limits 32Gi
+
+
+|^[4]^ Restic:
+
+CPU - Request 2000m, Limits 8000m
+
+
+Memory - Request 16Gi, Limits 40Gi
+
+
+|900m
+
+
+
+|^[5]^ DataMover
+|
+|
+|10m - average usage
+
+60m - large usage
+|===
+
+[.small]
+--
+1. Average usage - use these settings for most usage situations.
+
+2. Large usage - use these settings for large usage situations, such as a large PV (500GB Usage), multiple namespaces (100+), or many pods within a single namespace (2000 pods+), and for optimal performance for backup and restore involving large datasets.
+
+3. Restic resource usage corresponds to the amount of data, and type of data. For example, many small files or large amounts of data can cause Restic to utilize large amounts of resources.  Velero documentation references 500m as a supplied default, for most of our testing we found 200m request suitable with 1000m limit.  As cited in Velero documentation, exact CPU and memory usage is dependent on scale of files and directories, in addition to environmental limitations.
```

Comment:
`As cited in Velero documentation` should this have an article? 

Restic resource usage corresponds to the amount of data, and type of data. For example, many small files or large amounts of data can cause Restic to utilize a large amount of resources. The Velero documentation references 500m to be supplied as a default, for most of our testing we found a request of 200m  to be  suitable with 1000m limit. As cited in the Velero documentation, exact CPU and memory usage is dependent on the scale of files and directories, in addition to environmental limitations.

Reply:
I added a hyperlink to the relevant Velero documentation.